### DISCUSSION TITLE
P999 of write increases largely while Pulsar disk reads increase

### DISCUSSION THREAD

**Discussion by chrisxrepo** [apache/pulsar#20017] [Q&A]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### Version

v2.10.3 journal nvme + ledger hdd

### Minimal reproduce step

1. start to produde for some time
2. start to consume, producer and consumer coexistence
3. watch the monitor

### What did you expect to see?

produce latency not increass

### What did you see instead?

 pulsar produce monitor
<img width="1368" alt="image" src="https://user-images.githubusercontent.com/7782939/230062323-488fbdfa-d002-4f0d-a4ab-bbd8889c898c.png">

bookie monitor
<img width="1367" alt="image" src="https://user-images.githubusercontent.com/7782939/230062475-0d9467e6-ce3c-4c9e-bcad-6dee69325aac.png">
<img width="1357" alt="image" src="https://user-images.githubusercontent.com/7782939/230062552-35ceae04-6105-42c9-96bb-77d2f8f64176.png">

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by tisonkun** [comment]
This should be a use-case question. Moved to the Q&A forum.

Also, please translate the content to English for most community members reading your questions.

**Reply by chrisxrepo** [reply]
-done
